Visualizer/plotgame.py

```diff
@@ -1,16 +1,19 @@
 from pipper import install_packages
-install_packages(['pygame', 'euclid3'])
+install_packages(['pygame', 'numpy'])
 from rectangle import Rectangle
 from modules.CAN_Navigation.module.Algorithms import AStar
 from modules.CAN_Navigation.module.Structure import grid_factory
-import euclid3 as vmath
 import pygame as pg
+import numpy as np
 
 
-GRID_SIZE = 50
+GRID_SIZE = 60
 
 
 def main():
+    print("click to mark obstacles.")
+    print("S: mark start. \nE: mark end. \nSpace: plot path. \nC: clear all.")
+
     # grid size is fixed , tried 200x200 but it's really slow
```

Comment:
Either add why it becomes slow or redirect them to the wiki. I can't (logically) deduce why it would be slow based on this alone nor do I know if it can be fixed.

Reply:
added TODO that should clarify the issue.